src/common/storage/storageclasses/editorstorage.js

```diff
@@ -342,19 +347,39 @@ define([
             commitNext();
         };
 
-        this.makeCommit = function (projectId, branchName, parents, rootHash, coreObjects, msg, callback) {
+        this.makeCommit = function (projectId, branchName, parents, rootHash, dataObjects, msg, callback) {
```

Comment:
I see why you don't want it to be called coreObjects. However now there's an inconsistency between project api and this.